src/api/query.request.ts

```diff
@@ -1,19 +1,47 @@
 /* GraphQL queries */
-export const getAllDaoReq = (pageNumber: number, pageSize: number) => {
+export const getAllDaoReq = (
+	first: number | null,
+	after: string | null,
+	last: number | null,
+	before: string | null
```

Comment:
please add a doc comment about the parameters.
What's the difference between `first` and  `after`?

Reply:
Done